### DISCUSSION TITLE
Projection Manager

### DISCUSSION THREAD

**Discussion by StuartFergusonVme** [vmeretail/subscriptionservice#143] [Ideas]
Placeholder

**Comment by StevenBlair123** [comment]
Partitions - how will we manage this. This will directly related to how the state is loaded.
When using ES has persistence for state, make sure we are keeping only 1 event in the stream (metadata)
Discuss options for projection emitting to multiple handlers (what happens if one handler fails etc)
When we add partitions, allow the code to "GET" the state - might be useful for querying via REST
Partition - catchup is 1 event at time, but order guaranteed. if we used persistent subscriptions, we could be loading partitions concurrently, but order problems.

If we need to read from multiple streams, how is this handled?

**Comment by StevenBlair123** [comment]
Concurrency - what happens if we run multiple projections (the same one)
